regexes/client/libraries.yml

```diff
@@ -5,6 +5,21 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# got (https://github.com/sindresorhus/got)
+- regex: '^got '
+  name: 'got'
+  version: ''
+
+# Typhoeus (https://github.com/typhoeus/typhoeus)
+- regex: 'Typhoeus'
+  name: 'Typhoeus'
+  version: ''
+
+# ReqBin (https://reqbin.com/curl)
+- regex: 'ReqBin Curl Client/([\.\d]+)'
```

Comment:
I have not found a library that gives out such a user-agent.
I propose to transfer this definition of UA to bots.